R/get-path-from-object.R

```diff
@@ -131,6 +135,72 @@ get_model_id.bbi_nonmem_summary <- function(.mod) {
 }
 
 
+#' Get path to data file
+#'
+#' Helper to extract the path to the input data from a `bbi_nonmem_summary` or
+#' `bbi_nonmem_model` object. Note that **this function only works on models that
+#' have been successfully run** because the path to the data is constructed
+#' during model submission.
+#' @param .mod Model to use, either a `bbi_{.model_type}_model` or
+#' `bbi_{.model_type}_summary` object.
+#' @param ... Arguments passed through to methods.
+#'   (Currently none, but will likely have some modifier arguments for `bbi_stan_model` objects.)
+#' @importFrom fs path_rel
+#' @returns Absolute path to input data file
```

Comment:
What is the difference between `@return` and `@returns`?

Reply:
`@returns` is a typo while `@return` is the correct way to indicate the returned value. Nice catch.

Reply:
Seems like it's valid though;  the .Rd file was right. https://github.com/r-lib/roxygen2/issues/1148